api/app/wildfire_one/schema_parsers.py

```diff
@@ -163,61 +163,43 @@ def generate_station_daily(raw_daily,  # pylint: disable=too-many-locals
     isi = raw_daily.get('initialSpreadIndex', None)
     bui = raw_daily.get('buildUpIndex', None)
     ffmc = raw_daily.get('fineFuelMoistureCode', None)
-    fmc = cffdrs.foliar_moisture_content(
-        station.lat, station.long, station.elevation, get_julian_date_now())
+    cc = raw_daily.get('grasslandCuring')
+
+    # set default values in case the calculation fails (likely due to missing data)
+    fmc, sfc, ros, cfb, hfi, lb_ratio, bros, sixty_minute_fire_size, fire_type, intensity_group = None, None, None, None, None, None, None, None, None, None
     try:
+        fmc = cffdrs.foliar_moisture_content(
+            station.lat, station.long, station.elevation, get_julian_date_now())
         sfc = cffdrs.surface_fuel_consumption(
             FuelTypeEnum[fuel_type], bui, ffmc, pc)
-    except cffdrs.CFFDRSException:
-        sfc = None
-        logger.error('Encountered error when calculating surface_fuel_consumption for station %s', station.code)
-
-    cc = raw_daily.get('grasslandCuring')
-    if cc is None:
-        cc = FUEL_TYPE_DEFAULTS[fuel_type]["CC"]
-    try:
+        if cc is None:
+            cc = FUEL_TYPE_DEFAULTS[fuel_type]["CC"]
```

Comment:
Should we just set cc to this initially, so there's no need to set it to `None` in the first place?

Reply:
we don't exactly set `cc` to None initially; we pull the `grasslandCuring` value from `raw_daily` if it's available. We're just using the fuel type defaults if there's no constant value set for the specific station in WFWX